src/Directory.Build.props

```diff
@@ -16,14 +16,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
-    <PackageReadmeFile Condition="'$(IsPackable)' == 'true'">README.md</PackageReadmeFile>
-    <NoWarn>$(NoWarn);1591</NoWarn> <!--TODO: temporary solution-->
+    <IsPackable>true</IsPackable>
```

Comment:
> What problem did you solve moving properties around?

As one example, since IsPackable is true for all library projects by default, there was no way to set IsPackable to false for GraphQL.Dummy prior to the conditions within Directory.Build.props that checked IsPackable.

Once the conditions were moved to Directory.Build.targets, then those conditions were skipped for GraphQL.Dummy

There was probably similar issues with other projects.  After reading the documentation, I configured the settings in the correct places, and it works properly now.